twoliter/embedded/Makefile.toml

```diff
@@ -857,7 +857,7 @@ cargo build \
 ]
 
 [tasks.build-variant]
-dependencies = ["fetch", "build-sbkeys", "publish-setup", "validate-kits"]
+dependencies = ["fetch-sdk", "build-sbkeys", "publish-setup", "validate-kits"]
```

Comment:
`build-variant` can also trigger the equivalent of `build-package` tasks, and for those we do need to run `fetch-sources` (for first-party sources under `sources/`) and `fetch-vendored` (for packages that need bundled Go modules prepared).